src/main/java/io/github/thebusybiscuit/slimefun4/core/SlimefunRegistry.java

```diff
@@ -350,7 +350,7 @@ public Set<ItemHandler> getGlobalItemHandlers(@Nonnull Class<? extends ItemHandl
     }
 
     @Nonnull
-    public Map<String, BlockStorage> getWorlds() {
+    public Map<UUID, BlockStorage> getWorlds() {
```

Comment:
People rely on this method - https://github.com/search?q=BlockStorage+.getWorlds%28%29&type=code
This will break due to signature change.